lib/avromatic/messaging.rb

```diff
@@ -30,5 +30,30 @@ def decode(data, schema_name: nil, namespace: @namespace)
       reader = Avromatic::IO::DatumReader.new(writers_schema, readers_schema)
       reader.read(decoder)
     end
+
+    def encode(message, schema_name: nil, namespace: @namespace, subject: nil)
+      schema = @schema_store.find(schema_name, namespace)
+
+      # Schemas are registered under the full name of the top level Avro record
+      # type, or `subject` if it's provided.
+      schema_id = @registry.register(subject || schema.fullname, schema)
+
+      stream = StringIO.new
+
+      # The following line differs from the parent class to use a custom DatumWriter
+      writer = Avromatic::IO::DatumWriter.new(schema)
```

Comment:
I can. It's in its current location to more closely match the `avro_turf` source, but it doesn't matter much.